WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1009,12 +1006,9 @@ public void onStart() {
 
     @Override
     public void setTitle(@NonNull final String title) {
-        if (isAdded()) {
-            mToolbarTitle = (title.isEmpty()) ? getString(R.string.wordpress) : title;
-
-            if (mToolbar != null) {
-                mToolbar.setTitle(mToolbarTitle);
-            }
+        mToolbarTitle = title;
```

Comment:
I'm actually wondering if we need the body of this method at all since the toolbar title is a constant and we set it in the `onCreate` method